.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,72 @@
+# Preamble
+name: Deploy SaltProc docs
+
+on:
+  push:
+    branches:
+      - master 
+    paths:
+      - 'doc/**'
+      - 'saltproc/**'
+
+env:
+  CACHE_NUMBER: 0 #change to manually reset cache
+
+jobs:
+  deploy-docs:
+    runs-on: ubuntu-latest
+      #if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+      - uses: actions/checkout@v2
+
+# Setup conda 
+      - name: Set up conda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
+          miniforge-version: latest
+          activate-environment: saltproc-env
+          use-mamba: true
+          use-only-tar-bz2: true
+      - run: |
+          conda config --env --set pip_interop_enabled True
+
+# check for existing cache
+      - name: Set cache date
+        run: echo "DATE=$(date +'%Y%m%d')" >> $GITHUB_ENV
+
+      - uses: actions/cache@v2
+        id: dependencies-cache
+        with:
+          path: |
+            /usr/share/miniconda3/envs/saltproc-env
+            ~/.cache/pip
+          key: depcache-${{ hashFiles('environment.yml') }}-${{ env.DATE }}-${{ env.CACHE_NUMBER }}
+
+# Install dependencies
+      - name: Update environment
+        run: mamba env update -n saltproc-env -f environment.yml
+        if: steps.dependencies-cache.outputs.cache-hit != 'true'
+
+      - name: Install SaltProc
+        run:  pip install .
+
+      - name: Check packages
+        run: conda list
+
+      - name: Build Sphinx Documentation
+        run: |
+          cd doc
+          sphinx-apidoc --separate --force --output-dir=src/ ../saltproc
+          make clean
+          make html
```

Comment:
just to make sure since i don't see the `./src/` folder showing up in the makefile, do the apidocs generated by l64 show up in the local build?

Reply:
Oh I see. The src/ folder shows up in the index.rst file.

Reply:
There is a link to the /src/index.rst (the module code index) in the /index.rst (the base website index)